File: MODUL-4/python/PRAK402-2310817210007-RaymondHariyono.py
batas = int(input())
for i in range(1, batas+1):
    if i % 2 == 1:
        print(i, end=" ")
print()
for j in range(batas, 1, -1):
    if j % 2 == 0:
        print(j, end=" ")

File: MODUL-4/C/PRAK402-2310817210007-RaymondHariyono.c
#include <stdio.h>

int main (){
int batas;
scanf("%d",&batas);
for(int i= 1; i <= batas; i++){
    if (i % 2 == 1){
    printf("%d ", i);
    }
}
printf("\n");

for(int j = batas; j > 1; j--){
    if ( j % 2 == 0){
    printf("%d ", j);
    }
}
return 0;
}